srcpkgs/python3-youtube_search/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'python3-youtube_search'
+pkgname=python3-youtube_search
+version=1.1.0
+revision=1
+_gitver=86af3e73bd340f37be56f2be6a5ff706b16e454e
```

Comment:
pypi has versioned tarballs.

Reply:
It's an incomplete tarball, doesn't include the license

Reply:
Then download license separately. version= should be actually used.